src/interpreter/index.ts

```diff
@@ -113,7 +135,6 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
 
         interpreter.environment.nodeDefMap = componentMap;
 
-        let componentScopeResolver = new ComponentScopeResolver(componentMap, parseFn);
```

Comment:
Any reason why this was removed? We'd always have to pass an instance of `componentScopeResolver` with the same `componentMap`, right?